src/main/java/uk/ac/ebi/atlas/experimentpage/markergenes/CellTypeMarkerGene.java

```diff
@@ -0,0 +1,39 @@
+package uk.ac.ebi.atlas.experimentpage.markergenes;
+
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class CellTypeMarkerGene {
```

Comment:
Sure, I will make DTO generic to use for the cluster and cell type endpoints while working on the cluster endpoint task.

Reply:
I have modified variable names as "cellGroupType" and "cellGroupValue" throughout the code. Because those look more generic and more meaningful than previous.